source/_integrations/altruist.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: Altruist
+description: Instructions on how to setup Altruist Sensors in Home Assistant.
+ha_category:
+  - Health
+  - Sensor
+ha_config_flow: true
+ha_release: 2025.5
+ha_iot_class: Local Polling
+ha_codeowners:
+  - '@airalab'
+  - '@LoSk-p'
+ha_domain: altruist
+ha_platforms:
+  - sensor
+ha_integration_type: device
+ha_zeroconf: true
+---
+
+The Altruist integration connects Home Assistant to [Air Quality Sensor “Altruist“](https://robonomics.network/devices/altruist/)—device designed for decentralized environmental monitoring. It captures noise, dust, and temperature data from the sensor over HTTP, making it available as locally usable entities within Home Assistant.
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+IP Address:
+  description: "The local IP address for your Altruist device."
+{% endconfiguration_basic %}
+
+## Available sensors
+
+The integration will fetch data from each device. The following sensors are supported:
+
+- Humidity
+- Temperature
+- Atmospheric Pressure
```

Comment:
```suggestion
- Atmospheric pressure
```
we use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization)